build/bump_version.py

```
@@ -0,0 +1,112 @@
+        # 将字节转换为字符串进行处理
+        str_content = content.decode('utf-8')
+        pattern = r'(<parent>.*?<version>)(.*?)(</version>.*?</parent>)'
+        regex = re.compile(pattern, re.DOTALL)
+
+        if not regex.search(str_content):
+            print(f"Skipped {file_path} - no matching parent version found")
+            return
+
+        def replace_version(match):
+            old_version = match.group(2)
+            print(f"In {file_path}: Changing version from {
```

Pylint (3.3.7)

[error] Parsing failed: 'unterminated string literal (detected at line 42) (bump_version, line 42)'

(E0001)